OsmAnd-java/src/main/java/net/osmand/router/MissingMapsCalculator.java

```diff
@@ -23,10 +22,12 @@
 import net.osmand.map.OsmandRegions;
 import net.osmand.map.WorldRegion;
 import net.osmand.util.Algorithms;
+import net.osmand.util.CollectionUtils;
 import net.osmand.util.MapUtils;
 
 public class MissingMapsCalculator {
-	protected static final Log log = PlatformUtil.getLog(MissingMapsCalculator.class);
```

Comment:
Extra task to remove static:
```
public static void initMissingMapsCalculator(OsmandRegions osmandRegions) {
		if (missingMapsCalculator == null) {
			missingMapsCalculator = new MissingMapsCalculator(osmandRegions);
		}
	}
```
Idea: 
Add static OsmandRegions to PlatformUtil class, MissingMapsCalculator should be always initiated as used no static instance. 	
```
public MissingMapsCalculator() throws IOException {
```
Should be deleted